packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketOfferExchange/CoinmarketOfferExchangeSend.tsx

```diff
@@ -1,40 +1,101 @@
 
     return (
-        <Column gap={spacings.lg} alignItems="stretch" flex="1">
-            <InfoItem label={<Translation id="TR_EXCHANGE_SEND_FROM" />} typographyStyle="hint">
-                <AccountLabeling account={account} />
-            </InfoItem>
-            <InfoItem
-                label={<Translation id="TR_EXCHANGE_SEND_TO" values={{ providerName }} />}
-                typographyStyle="hint"
-            >
-                {sendAddress}
-            </InfoItem>
-            <Column margin={{ top: 'auto' }}>
-                <Divider margin={{ top: spacings.xs, bottom: spacings.lg }} />
-                <Button
-                    data-testid="@coinmarket/offer/exchange/confirm-on-trezor-and-send"
-                    isLoading={callInProgress}
-                    onClick={sendTransaction}
-                >
-                    <Translation id="TR_EXCHANGE_CONFIRM_ON_TREZOR_SEND" />
-                </Button>
-            </Column>
-        </Column>
+        <>
+            {(status === 'CONFIRM' || status === 'SENDING') && sendAddress ? (
+                <Column gap={spacings.lg} alignItems="stretch" flex="1">
+                    <InfoItem
+                        label={<Translation id="TR_EXCHANGE_SEND_FROM" />}
+                        typographyStyle="hint"
+                    >
+                        <AccountLabeling account={account} />
+                    </InfoItem>
+                    <InfoItem
+                        label={<Translation id="TR_EXCHANGE_SEND_TO" values={{ providerName }} />}
+                        typographyStyle="hint"
+                    >
+                        {sendAddress}
+                    </InfoItem>
+                    <Column margin={{ top: 'auto' }}>
+                        <Divider margin={{ top: spacings.xs, bottom: spacings.lg }} />
+                        <Button
+                            data-testid="@coinmarket/offer/exchange/confirm-on-trezor-and-send"
+                            isLoading={callInProgress}
+                            onClick={sendTransaction}
+                        >
+                            <Translation id="TR_EXCHANGE_CONFIRM_ON_TREZOR_SEND" />
+                        </Button>
+                    </Column>
+                </Column>
+            ) : (
+                <WaitingWrapper>
+                    <Spinner />
+                    <Title>
+                        <Translation
+                            id="TR_SELL_DETAIL_WAITING_FOR_SEND_CRYPTO"
+                            values={{ providerName }}
+                        />
+                    </Title>
+                    <Value>
+                        <Translation
+                            id="TR_SELL_DETAIL_WAITING_FOR_SEND_CRYPTO_INFO"
+                            values={{ providerName }}
+                        />
+                    </Value>
+                </WaitingWrapper>
```

Comment:
Then we can delete all custom styles. We don't need bold text, I guess.

```suggestion
            <Column
                justifyContent="center"
                margin={{ horizontal: spacings.lg, vertical: spacings.xxxxl }}
                gap={spacings.xl}
            >
                <Spinner />
                <Column>
                    <Text>
                        <Translation
                            id="TR_SELL_DETAIL_WAITING_FOR_SEND_CRYPTO"
                            values={{ providerName }}
                        />
                    </Text>
                    <Text variant="tertiary">
                        <Translation
                            id="TR_SELL_DETAIL_WAITING_FOR_SEND_CRYPTO_INFO"
                            values={{ providerName }}
                        />
                    </Text>
                </Column>
            </Column>
```

Reply:
its copy from sell, we can do some refactor later

Reply:
If the part of the code is the same, make it a reusable component. I also noticed that `TR_SELL_DETAIL_WAITING_FOR_SEND_CRYPTO_INFO` could be renamed to `TR_COINMARKET_DETAIL_WAITING_FOR_SEND_CRYPTO_INFO`

The current state is the same parts of the same code using custom styles. Please 🙏